lib/src/QGVMapQGView.cpp

```diff
@@ -599,3 +602,27 @@ void QGVMapQGView::keyPressEvent(QKeyEvent* event)
 {
     QWidget::keyPressEvent(event);
 }
+
+void QGVMapQGView::dragEnterEvent(QDragEnterEvent *event)
+{
+    event->accept();
+}
+
+void QGVMapQGView::dragMoveEvent(QDragMoveEvent *event)
+{
+    event->accept();
+}
+
+void QGVMapQGView::dropEvent(QDropEvent *event)
+{
+    if (event->mimeData()->hasFormat("application/x-qabstractitemmodeldatalist"))
+    {
+        emit dropData(event->position(), event->mimeData());
```

Comment:
The first thing I tried was to subclass QGVMap to add the accept drop functionality, because I didn't want to change your library. However QGVMap inherits from QWidget. It looks that QWidget doesn't send the drop event to a child QGraphicsView inside of it. I think this is a limitation of QWidget or QGraphicsView, not sure (maybe this is the correct behavior). 
Then I realized that I would need to change QGVMapQGView class to achieve the intended behavior. 

Well... I didn't restricted the mime types, so there is a plenty of freedom for new implementations.

Reply:
I was wrong. I really restricted the type of mime-data on the QGVMapQGView class. Sorry. I just fixed all the remaining errors.

Reply:
Well, I did my best to fix clang-forma error on pre-commit. Unfortunately, I could not fix it locally.